src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -113,6 +119,9 @@ def validate_port(port, force_start):
         else:
             app.logger.setLevel(logging.INFO)
         print(f"Start Prompt Flow Service on http://localhost:{port}, version: {get_promptflow_sdk_version()}")
+        app.logger.info(
```

Comment:
add comment on why we need two here. consider warp a function here

Reply:
Remove print message for msi, it can get app since using sdk to start waitress. 

wrap message for python case